fpsim/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.26.7'
-__versiondate__ = '2023-10-24'
+__version__ = '0.26.8'
```

Comment:
what would it take to get to 1.0? 😮

Reply:
That's definitely a @MObrien-IDM question :grin: